27-cellular-connected-electronic-kiosk/README.md

```diff
@@ -0,0 +1,138 @@
+
+#### `kiosk_content_version`
+
+This is a version number for the file specified by `kiosk_content`. If you update the content, you can increment this number to cause the Python script to download and use the new content.
+
+#### `kiosk_download_time`
+
+This variable specifies the hour of the day (using 24-hour time) when the Python script should check for updates to the `kiosk_content`. If, for example, `kiosk_download_time` is set to 9, the script will check for updates to the content at 9:00 AM local time. If the content file name or version has changed, the script will download and use the new content. If this variable is set to the special value `now`, any change to the content file name or version will cause an immediate download of the new content, instead of waiting for a specific hour of the day.
+
+#### `kiosk_data`
+
+This is an optional environment variable that can be set to any valid JSON object. It can be used to overwrite the contents of the `data.js` file in the ZIP. For instance, if the contents of `data.js` looks like this:
+
+```javascript
+var data = <value of kiosk_data goes here>
+```
+
+You can set the `kiosk_data` JSON to `{"message":"Hello world!"}`, and the Python script will translate it into:
+
+```javascript
+var data = {"message":"Hello world!"}
+```
+
+For use in the web app. The web application can then access and use this data however you want. In this way, `kiosk_data` acts as a way of passing dynamic data to the web app without needing to modify the contents of the ZIP file and re-download the whole thing.
+
+To see this in action with actual code examples, see the [related sample repo](https://github.com/blues/accelerators-cellular-connected-electronic-kiosk).
+
+![Environment variables](images/env_vars.png)
+
+## Hardware
+
+The following hardware is required to run the Cellular-Connected Electronic Kiosk project.
+
+- [Raspberry Pi 4B](https://www.raspberrypi.com/products/raspberry-pi-4-model-b/)
+- [CanaKit 3.5A Raspberry Pi 4 Power Supply (USB-C)](https://www.amazon.com/CanaKit-Raspberry-Power-Supply-USB-C/dp/B07TYQRXTK/)
+- microSD Card (8GB minimum) and a MicroSD to SD Memory Card Adapter (your development PC must have an SD card reader slot)
+- [Notecard](https://shop.blues.io/collections/notecard/products/note-wbna-500)
+- [Notecarrier Pi Hat](https://shop.blues.io/products/carr-pi)
+- Micro USB cable
+- [7" Touchscreen Display for Raspberry Pi](https://www.adafruit.com/product/2718)
+- [Wireless Keyboard and Mouse Combo](https://www.adafruit.com/product/1738)
+- [SmartiPi Touch Pro for Raspberry Pi and Official 7" Display](https://www.adafruit.com/product/4951)
+
+Once you have all your hardware you'll need to install the Raspberry Pi OS and assemble the pieces.
```

Comment:
I would insert a line break after this line before the next heading.

Reply:
Agreed